clap_man/src/render.rs

```diff
@@ -0,0 +1,195 @@
+use std::io::Write;
+
+use crate::man::{bold, document_title, escape, indent, italic, list, section_title};
+
+use clap::ArgSettings;
+
+fn app_name(app: &clap::App) -> String {
+    if let Some(name) = app.get_bin_name() {
+        escape(name)
+    } else {
+        escape(app.get_name())
+    }
```

Comment:
I suspect we should use `get_name`.

In the case someone has previously parsed with this `App`, `get_bin_name` will be set to `argv[0].file_name()`.  So whatever someone has this linked to is what will get help.  Additionally, on Windows, it will include `.exe`, though we are thinking of removing that.

Reply:
Good catch, I've updated to just use `get_name` everywhere.